src/pages/wallet-overview/wallet-overview.ts

```diff
@@ -19,7 +21,15 @@ import { ITransaction } from "../../models/ITransaction";
  * Ionic pages and navigation.
  */
 
-export declare type VisibilityType = "shown" | "hidden"; 
+export declare type VisibilityType = "shown" | "hidden";
+
+declare let cordova: any;
```

Comment:
Is this declaration used? Usually you don't want to communicatie with Cordova in a controller directly. Instead take a look at how Ionic Native wrap the Bluetooth Low Energy functionality [here](https://github.com/ionic-team/ionic-native/blob/master/src/%40ionic-native/plugins/ble/index.ts). Ionic has support for easily wrapping Cordova functionality.

Reply:
Cordova File Native directives used next request